dotnetcore3.1/cookiecutter-aws-sam-quick-start-s3-dotnet/{{cookiecutter.project_name}}/events/event.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
The event file doesn't look like the one from nodejs one, is this a different event?
https://github.com/aws/aws-sam-cli-app-templates/blob/master/nodejs14.x/cookiecutter-quick-start-s3/%7B%7Bcookiecutter.project_name%7D%7D/events/event-s3.json

Reply:
@mndeveci nice catch.